Doc/library/stdtypes.rst

```diff
@@ -5236,9 +5238,11 @@ instantiated from the type::
    TypeError: cannot create 'types.UnionType' instances
 
 .. note::
-   The :meth:`__or__` method for type objects was added to support the syntax
-   ``X | Y``.  If a metaclass implements :meth:`__or__`, the Union may
-   override it::
+   The :meth:`!__or__` method for type objects was added to support the syntax
+   ``X | Y``.  If a metaclass implements :meth:`!__or__`, the Union may
```

Comment:
I considered linking to https://docs.python.org/3/reference/datamodel.html#object.__or__, but the data model only discusses implementing `__or__` in the context of emulating numeric types, and that's pretty distinct to the purpose of `type.__or__`